src/layer/GeoJSON.js

```diff
@@ -240,24 +240,25 @@ export function coordsToLatLngs(coords, levelsDeep, _coordsToLatLng) {
 	return latlngs;
 }
 
-// @function latLngToCoords(latlng: LatLng): Array
+// @function latLngToCoords(latlng: LatLng, precision?: Number): Array
 // Reverse of [`coordsToLatLng`](#geojson-coordstolatlng)
-export function latLngToCoords(latlng) {
+export function latLngToCoords(latlng, precision) {
+	precision = precision || 6;

```

Comment:
Oh, good catch, I'll update that. I'm going to leave the default of `6` for now, though, because I don't know when/if your PR will come through.